zc_events/exceptions.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from django.core.exceptions import ImproperlyConfigured
+    from django.core.exceptions import ImproperlyConfigured  # pylint: disable=unused-import
```

Comment:
I'm not sure why this was necessary, but I think we can just import this without having the `except` case once we update our django version.